### ISSUE TITLE
// #endif is invalid unless it is followed by a new line

### ISSUE THREAD

**Issue by dahmian** [dcodeIO/Preprocessor.js#2]
When I tried to run the following example from the readme, I get an error:

``` javascript
// #if 1==2
console.log("1==2");
// #elif 2==2
console.log("2==2");
// #endif
```

Error: Illegal #endif: // #endif...
    at Preprocessor.process (/usr/local/lib/node_modules/preprocessor/Preprocessor.js:258:31)

But if I add in a new line after the #endif, then it works:

``` javascript
// #if 1==2
console.log("1==2");
// #elif 2==2
console.log("2==2");
// #endif

```

It looks like the unit test is also expecting a \n, so that's why it passes. https://github.com/dcodeIO/Preprocessor.js/blob/master/tests/suite.js#L33

**Comment by dcodeIO** [comment]
Thanks for reportings this! The issue has been fixed in 1.0.3 (also on NPM now).

**Comment by dahmian** [comment]
Awesome, thanks for fixing it! I downloaded it from NPM and verified that it works.
